cnd/src/btsieve/client.rs

```diff
@@ -214,13 +222,51 @@ impl BtsieveHttpClient {
         Box::new(
             self.client
                 .delete(query.as_ref().clone())
+                .headers(construct_headers())
                 .send()
                 .map(|_| ())
                 .map_err(|e| {
                     Error::FailedRequest(format!("Failed to delete query because {:?}", e))
                 }),
         )
     }
+
+    pub fn health(&self) -> Box<dyn Future<Item = (), Error = ()> + Send> {
+        let mut url = self.endpoint.clone();
+        url.set_path("health");
```

Comment:
Considering that we are not sticking to the health RFC, shouldn't we call it something else than `/health` ?

Reply:
No strong opinion on this, so happy to change it.